### PR TITLE
update name

### PR DESCRIPTION
**Pull request by declmal**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers).

### CODE REVIEW

File: cvm/quantization/tfm_ops.py [hunk 1]
```diff
@@ -152,9 +156,9 @@ def _fuse_bias(self, op, infer_shapes):
         X, W, B = childs
         oshp = infer_shapes[op.attr('name')][0]
         op = mx.sym.Convolution(X, W, **attr, name=op.attr('name'))
```
Comment (wlt-cortex) [inline cvm/quantization/tfm_ops.py]: use N.n

File: cvm/quantization/tfm_ops.py [hunk 2]
```diff
@@ -329,12 +331,12 @@ def rewrite(self, op, **kwargs):
             X = childs[0]
             X_name = X.attr('name')
             X_shape = infer_shapes[X_name][get_entry_id(X)]
-            scale_name = X_name + '_avg_scale'
+            scale_name = N.n('avg_scale')
             assert scale_name not in graph
```
Comment (wlt-cortex) [inline cvm/quantization/tfm_ops.py]: delete

Comment (declmal) [reply cvm/quantization/tfm_ops.py]: 'assert' line has been deleted

File: cvm/quantization/tfm_ops.py (hunk 1 above)
Comment (declmal) [reply cvm/quantization/tfm_ops.py]: fixed

**Pull request merged**